junit-jupiter-api/src/main/java/org/junit/jupiter/api/condition/EnabledIf.java

```diff
@@ -116,6 +116,7 @@
  * this annotation may be used in conjunction with other {@code @Enabled*} or
  * {@code @Disabled*} annotations in this package.
  *
+ * @deprecated for removal
```

Comment:
We should add a note about what to do instead here and for `@DisabledIf`.

Reply:
Good idea. Will add a sentence here and there.

Reply:
What about?

```
Script-based conditions APIs, and its supporting implementations
are deprecated with the intent to remove them in JUnit Jupiter 5.6.
Users should instead create and use a dedicated implementation of
{@link org.junit.jupiter.api.extension.ExecutionCondition ExecutionCondition}.
```

Reply:
> Script-based condition APIs and their supporting implementations are deprecated with the intent to remove them in JUnit Jupiter 5.6. Users should instead rely on a combination of other built-in conditions or create and use a custom implementation of {@link org.junit.jupiter.api.extension.ExecutionCondition ExecutionCondition} to evaluate the same conditions.